### DISCUSSION TITLE
How do I include the name of the problem in the filename?

### DISCUSSION THREAD

**Discussion by markjouh** [cpeditor/cpeditor#1181] [Q&A] [answered]
I'm moving to cpeditor from vscode, where I used the cph (competitive programming helper) extension. Upon parsing a task using competitive companion, it would create a file with a name like "D_Salary_Changing.cpp". However, in cpeditor, it simply names it "D.cpp". How can I configure cpeditor to automatically include the full name of the problem like cph? Thanks!

**Comment by ouuan** [comment, marked as answer]
Now you can't. There is issue #580 for this function.
